Description: Remove AM_C_PROTOTYPES and ansi2knr from autoconf files.
Debian-Bug: http://bugs.debian.org/713673
Author: Barry deFreese <bdefreese@debian.org>
Index: libcdaudio-0.99.12p2/configure.ac
===================================================================
--- libcdaudio-0.99.12p2.orig/configure.ac	2013-07-02 07:44:09.000000000 -0400
+++ libcdaudio-0.99.12p2/configure.ac	2013-07-02 07:44:09.000000000 -0400
@@ -17,7 +17,6 @@
 AC_PROG_AWK
 AC_PROG_CC
 AC_PROG_CPP
-AM_C_PROTOTYPES
 AM_PROG_CC_STDC
 AC_PROG_INSTALL
 AC_PROG_LN_S
Index: libcdaudio-0.99.12p2/Makefile.am
===================================================================
--- libcdaudio-0.99.12p2.orig/Makefile.am	2004-10-02 13:50:59.000000000 -0400
+++ libcdaudio-0.99.12p2/Makefile.am	2013-07-02 07:44:26.000000000 -0400
@@ -1,5 +1,4 @@
 ## Process this file with automake to produce Makefile.in
-AUTOMAKE_OPTIONS = ansi2knr
 SUBDIRS = src
 
 bin_SCRIPTS = libcdaudio-config
